pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -24,6 +25,10 @@ import (
 
 var log = logf.Log.WithName("controller_unifiedpushserver")
 
+var (
```

Comment:
Would it make sense to include the `log` variable on the line above in this `var ()` block?

Reply:
+1